draft-packages/illustration/KaizenDraft/Illustration/Spot.tsx

```diff
@@ -725,6 +841,167 @@ export const OneOnOne = ({ enableAspectRatio, ...props }: SpotProps) => (
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-community.svg"
+  />
+)
+export const Company = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-company.svg"
+  />
+)
+export const Conversations = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-conversations.svg"
+  />
+)
+export const DataVisualization = ({
+  enableAspectRatio,
+  ...props
+}: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-data-visualization.svg"
+  />
+)
+export const Goals = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-goals.svg"
+  />
+)
+export const Insights = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-insights.svg"
+  />
+)
+export const Learn = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-learn.svg"
+  />
+)
+export const Microphone = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-microphone.svg"
+  />
+)
+export const PaperPen = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-paper-pen.svg"
+  />
+)
+export const PowerfulInsights = ({
+  enableAspectRatio,
+  ...props
+}: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-powerful-insights.svg"
+  />
+)
+export const Privacy = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-privacy.svg"
+  />
+)
+export const Process = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-process.svg"
+  />
+)
+export const Resources = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-resources.svg"
+  />
+)
+
+// TODO: Compare with Covid-19 Response spot
```

Comment:
Do you still need this?

Reply:
Good find - I will fix that up

Reply:
@lloydstubber - I have updated it to remove as it was a duplicate illustration.